cypress/integration/fee-fine/newFeeFines.spec.js

```diff
@@ -11,108 +12,186 @@ import UsersCard from '../../support/fragments/users/usersCard';
+              testData.userServicePoint = points.body.defaultServicePointId;
+            });
             UsersOwners.createViaApi({ owner: uuid() }).then(owner => {
               testData.owner = { id : owner.id, name : owner.ownerName };
               ManualCharges.createViaApi({ ...ManualCharges.defaultFeeFineType, ownerId: owner.id }).then(manualCharge => {
                 testData.feeFineType = { id:  manualCharge.id, feeFineTypeName: manualCharge.feeFineType };
                 PaymentMethods.createViaApi(testData.owner.id).then(createdPaymentMethod => {
                   testData.paymentMethod = { id: createdPaymentMethod.id, name: createdPaymentMethod.name };
-                  // TODO: clarify why initial login is needed to load accordion with Fee/Fines
-                  // cy.loginAsAdmin({ path: AppPaths.getUserPreviewPath(testData.userId), waiter: UsersCard.waitLoading });
+                  // providing path in the login function doesn't working
+                  // TODO: investigate
                   cy.loginAsAdmin();
                   cy.visit(AppPaths.getUserPreviewPath(testData.userProperties.id));
                   UsersCard.waitLoading();
                 });
               });
             });
+
+            cy.getMaterialTypes({ limit: 1 }).then((res) => { testData.materialType = res.id; });
+            cy.getLocations({ limit: 1 }).then((res) => { testData.location = res.id; });
+            cy.getHoldingTypes({ limit: 1 }).then((res) => { testData.holdingType = res[0].id; });
+            InventoryHoldings.getHoldingSources({ limit: 1 }).then((res) => { testData.holdingSource = res[0].id; });
+            cy.getInstanceTypes({ limit: 1 }).then((res) => { testData.instanceType = res[0].id; });
+            cy.getLoanTypes({ limit: 1 }).then((res) => { testData.loanType = res[0].id; });
+          }).then(() => {
+            cy.createInstance({
+              instance: {
+                instanceTypeId: testData.instanceType,
+                title: testData.instanceTitle,
+              },
+              holdings: [{
+                holdingsTypeId: testData.holdingType,
+                permanentLocationId: testData.location,
+                sourceId: testData.holdingSource,
+              }],
+              items: [
+                [{
+                  barcode: ITEM_BARCODE,
+                  missingPieces: '3',
+                  numberOfMissingPieces: '3',
+                  status: { name: 'Available' },
+                  permanentLoanType: { id: testData.loanType },
+                  materialType: { id: testData.materialType },
+                }],
+              ],
+            });
           });
       });
     });
   });
 
   it('C455 Verify "New fee/fine" behavior when "Charge & pay now" button pressed (spitfire)', { tags: [TestType.smoke, Features.feeFine, devTeams.firebird] }, () => {
```

Comment:
change devTeams to spitfire please
![image](https://user-images.githubusercontent.com/26041116/180142291-b0b166d3-464b-4fc2-9bdd-14da36447516.png)

Reply:
done